broker/src/main/java/org/apache/rocketmq/broker/BrokerController.java

```diff
@@ -1025,9 +1025,7 @@ public void registerProcessor() {
         replyMessageProcessor.registerSendMessageHook(sendMessageHookList);
 
         this.remotingServer.registerProcessor(RequestCode.SEND_REPLY_MESSAGE, replyMessageProcessor, replyMessageExecutor);
-        this.remotingServer.registerProcessor(RequestCode.SEND_REPLY_MESSAGE_V2, replyMessageProcessor, replyMessageExecutor);
```

Comment:
Why is the `SEND_REPLY_MESSAGE_V2` being removed? Will this cause any compatibility issues?

Reply:
there may compatibility issues, A better way is to add a new one SEND_REPLY_MESSAGE_V3 message code

Reply:
@aaron-ai hi, I have optimized the compatibility issue, Preserved `SEND_ REPLY_ MESSAGE` and `SEND_ REPLY_ MESSAGE_ V2` request code And the corresponding `processor`